install/share/ipa.conf.template

```diff
@@ -100,6 +100,13 @@ WSGIScriptReloading Off
 # Target for login with internal connections
 Alias /ipa/session/cookie "/usr/share/ipa/gssapi.login"
 
+# Turn off Apache authentication for i18n messages
+<Location "/ipa/i18n_messages">
+  <RequireAll>
+    Require method POST
```

Comment:
Why did you choose this way to change authentication and why only ``POST``? There are many more HTTP verbs that can modify data, e.g. ``DELETE`` or ``PATCH``.

Reply:
If you mean why not:
```apache
<Location "/ipa/i18n_messages">
  Satisfy Any 
  Order Deny,Allow
  Allow from all 
</Location>
```
From docs (https://httpd.apache.org/docs/2.4/howto/access.html):
 ```apache
The Allow, Deny, and Order directives, provided by mod_access_compat, are deprecated and will go away in a future version. You should avoid using them, and avoid outdated tutorials recommending their use.
```
For this case the only allowed HTTP method is "POST" ( for strict configuration), all the others are refused. The goal of this endpoint is process of "i18n_messages" JSON requests via "POST".

Reply:
I've changed apache config to allow all unconditionally.
```apache
<Location "/ipa/i18n_messages">
   Require all granted
</Location>
```